### PR TITLE
Add gfx1151 to 6.2

### PR DESCRIPTION
**Pull request by amd-garydeng**
Summary of proposed changes:
add support for gfx1151 into 6.2

### CODE REVIEW

**Review by TorreZuk — APPROVED** [review]
LGTM for local changes but tensile tag is tip of your current Tensile PR so must wait for Tensile PR merge and update accordingly
Andrew should also approve this.

**Review by TorreZuk — COMMENTED** [review]
Can you also update the VERSION_STRING in root CMakeLists.txt, right @amcamd  ?

**Review by TorreZuk — COMMENTED** [review]
Tensile merge first then update tag commit, then I approve

File: CMakeLists.txt [hunk 1]
```diff
@@ -100,17 +100,17 @@ get_os_id(OS_ID)
...
 list( APPEND CMAKE_PREFIX_PATH ${ROCM_PATH}/llvm ${ROCM_PATH} ${ROCM_PATH}/hip /opt/rocm/llvm /opt/rocm /opt/rocm/hip )
 
 # setting target list based on ROCm version
-set( TARGET_LIST_ROCM_5.6 "gfx803;gfx900;gfx906:xnack-;gfx908:xnack-;gfx90a:xnack+;gfx90a:xnack-;gfx1010;gfx1012;gfx1030;gfx1100;gfx1101;gfx1102")
-set( TARGET_LIST_ROCM_5.7 "gfx803;gfx900;gfx906:xnack-;gfx908:xnack-;gfx90a:xnack+;gfx90a:xnack-;gfx940;gfx941;gfx942;gfx1010;gfx1012;gfx1030;gfx1100;gfx1101;gfx1102")
-set( TARGET_LIST_ROCM_6.0 "gfx900;gfx906:xnack-;gfx908:xnack-;gfx90a:xnack+;gfx90a:xnack-;gfx940;gfx941;gfx942;gfx1010;gfx1012;gfx1030;gfx1100;gfx1101;gfx1102")
+set( TARGET_LIST_ROCM_5.6 "gfx803;gfx900;gfx906:xnack-;gfx908:xnack-;gfx90a:xnack+;gfx90a:xnack-;gfx1010;gfx1012;gfx1030;gfx1100;gfx1101;gfx1102;gfx1151")
+set( TARGET_LIST_ROCM_5.7 "gfx803;gfx900;gfx906:xnack-;gfx908:xnack-;gfx90a:xnack+;gfx90a:xnack-;gfx940;gfx941;gfx942;gfx1010;gfx1012;gfx1030;gfx1100;gfx1101;gfx1102;gfx1151")
+set( TARGET_LIST_ROCM_6.0 "gfx900;gfx906:xnack-;gfx908:xnack-;gfx90a:xnack+;gfx90a:xnack-;gfx940;gfx941;gfx942;gfx1010;gfx1012;gfx1030;gfx1100;gfx1101;gfx1102;gfx1151")
```
Comment (amcamd) [inline CMakeLists.txt:115]: Is gfx1151 needed in: 
- TARGET_LIST_ROCM_5.7
- TARGET_LIST_ROCM_5.6

I expect it is only needed for TARGET_LIST_ROCM_6.0. I do not think the compiler for the older versions of ROCm have support for gfx1151.

**Review by amcamd — COMMENTED** [review]
The change to VERSION_STRING and CHANGELOG.md look good.

File: CMakeLists.txt (hunk 1 above)
Comment (TorreZuk) [reply CMakeLists.txt:115]: Agree they should only go into 6.0, but even there it is slightly incorrect.   The proper thing would be a new TARGET_LIST_ROCM_6.2.4  
elseif(${ROCM_PLATFORM_VERSION} VERSION_LESS 6.2.4)
set(SUPPORTED_TARGETS "${TARGET_LIST_ROCM_6.0}")
else....  

@amd-garydeng  can you add that change too please
...

File: CMakeLists.txt [hunk 2]
```diff
@@ -111,14 +111,17 @@ list( APPEND CMAKE_PREFIX_PATH ${ROCM_PATH}/llvm ${ROCM_PATH} ${ROCM_PATH}/hip /
 set( TARGET_LIST_ROCM_5.6 "gfx803;gfx900;gfx906:xnack-;gfx908:xnack-;gfx90a:xnack+;gfx90a:xnack-;gfx1010;gfx1012;gfx1030;gfx1100;gfx1101;gfx1102")
 set( TARGET_LIST_ROCM_5.7 "gfx803;gfx900;gfx906:xnack-;gfx908:xnack-;gfx90a:xnack+;gfx90a:xnack-;gfx940;gfx941;gfx942;gfx1010;gfx1012;gfx1030;gfx1100;gfx1101;gfx1102")
 set( TARGET_LIST_ROCM_6.0 "gfx900;gfx906:xnack-;gfx908:xnack-;gfx90a:xnack+;gfx90a:xnack-;gfx940;gfx941;gfx942;gfx1010;gfx1012;gfx1030;gfx1100;gfx1101;gfx1102")
+set( TARGET_LIST_ROCM_6.2.4 "gfx1151")
```
Comment (TorreZuk) [inline CMakeLists.txt]: This should be the full 6.0 list with ;gfx1151 added to end.

**Review by TorreZuk — APPROVED** [review]
LGTM thanks

**Review by amcamd — APPROVED** [review]

**Review by yoichiyoshida — APPROVED** [review]

**Review by amcamd — APPROVED** [review]
Tensile tag updated to tip of release/rocm-rel-6.2 on Oct 2   `81ae9537671627fe541332c0a5d953bfd6af71d6`

**Pull request merged**